documentation/TechnicalDesignDocument.tex

```diff
@@ -45,6 +45,25 @@
     %------------------------------------------------------------------------------------------------------------------------------------------------------- Requirements
     \section{Requirements}
     Functional and Non-functional requirements.
+    \subsection{Functional Requirements:}
+    \begin{itemize}
+        \item The program should support loading different Marama shapes using the Marama transpiler.
```

Comment:
Transpiler builds a file that specifies multiple maramafications. The program only supports loading .mar files (which are, at this moment, still json files, but should contain object data at a later date).

Reply:
Which the transpiler creates, yes?
And this is about what the program *should* do, not what it does now.

Reply:
You are right